components/script_traits/lib.rs

```diff
@@ -177,7 +177,7 @@ pub enum ConstellationControlMsg {
     /// Notifies script thread that a url should be loaded in this iframe.
     Navigate(PipelineId, SubpageId, LoadData),
     /// Requests the script thread forward a mozbrowser event to an iframe it owns
```

Comment:
Make sure to update all the doc comments for things you change to explain what happens if the subpage id is none.